### PR TITLE
[RW-6988][risk=low] POST -> GET

### PR DESCRIPTION
**Pull request by dmohs**
Does not modify data, so request should be a GET when feasible.

https://precisionmedicineinitiative.atlassian.net/browse/RW-6988

Existing:
```
curl -H "Authorization: Bearer $DMFBT" 'localhost:8081/v2/workspaces/aou-rw-test-53ff4756/mohstest/resources?resourceTypesToFetch=COHORT'
[]
```

New:
```
% curl -H "Authorization: Bearer $DMFBT" 'localhost:8081/v2/workspaces/aou-rw-test-53ff4756/mohstest/resources?resourceTypesToFetch=COHORT,DATASET'
[]
```

---
**PR checklist**

- [x] This PR meets the Acceptance Criteria in the JIRA story
- [x] The JIRA story has been moved to Dev Review
- [x] This PR includes appropriate unit tests
- [x] I have added explanatory comments where the logic is not obvious
- [x] I have run and tested this change locally, and my testing process is described here
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later
- [ ] If this includes an API change, I have run the E2E tests on this change against my local server with [yarn test-local](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples) because this PR won't be covered by the CircleCI tests 
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P)
- [ ] If this change impacts deployment safety (e.g. removing/altering APIs which are in use) I have documented these in the description
- [x] If this includes an API change, I have updated the appropriate Swagger definitions and updated the appropriate API consumers
  * AoU UI
  * [Perf tests](https://github.com/broadinstitute/mcnulty/blob/develop/src/test/scala/services/AoU.scala)
  * [Researcher Directory export](https://github.com/all-of-us/workbench/wiki/Researcher-Directory-(RDR-export))
  * Cron tasks - for Offline*Controllers
  * SumoLogic - for EgressAlert

### CODE REVIEW

File: api/db/local-vars.env [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 DB_DRIVER=com.mysql.jdbc.Driver
 # "db" hostname is established for the local docker network via docker-compose.yaml
-DB_HOST=db
+DB_HOST="${DB_HOST:-db}"
```
Comment (dmohs) [inline api/db/local-vars.env:6]: This mean, set to the value "db" only if the variable hasn't been set already.

File: api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java [hunk 2]
```diff
@@ -871,9 +872,21 @@ public ResponseEntity<RecentWorkspaceResponse> updateRecentWorkspaces(
 
   @Override
   public ResponseEntity<WorkspaceResourceResponse> getWorkspaceResources(
-      String workspaceNamespace,
-      String workspaceId,
-      WorkspaceResourcesRequest workspaceResourcesRequest) {
+      String ns, String id, WorkspaceResourcesRequest wrr) {
+    return getWorkspaceResourcesImpl(ns, id, wrr.getTypesToFetch());
```
Comment (dmohs) [inline api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java:876]: The shortened variable names help to make this line stand out, which is the only bit of real work done in this method.

File: api/src/main/resources/workbench-api.yaml [hunk 3]
```diff
@@ -1042,7 +1042,29 @@ paths:
       operationId: getWorkspaceResources
       responses:
         200:
-          description: Returns all objects that describe a section of the CDR in a workspace.
+          description: OK
```
Comment (dmohs) [inline api/src/main/resources/workbench-api.yaml:1045]: This is the successful response. The endpoint description (4 lines up) should say what this returns. In my opinion, this should be omitted except in unusual cases, but it is required.

File: e2ev2/README.md [hunk 4]
```diff
@@ -35,6 +35,18 @@ Full suite:
...
+## Useful Resources
+
+Jest's `expect` global provides assertions:
+
+https://jestjs.io/docs/expect
+
+Options for `yarn test ...`:
+
+https://jestjs.io/docs/cli
+
```
Comment (dmohs) [inline e2ev2/README.md:47]: Unrelated doc updates.

**Review by jmthibault79 — APPROVED** [review]
Looks good, thanks.

Please also run a `./gradlew spotlessApply` and update the UI code so it references the new endpoint instead of the old.

**Pull request merged**